stl/inc/yvals_core.h

```diff
@@ -526,14 +526,7 @@
 #define _STL_STRINGIZE_(S) #S
 #define _STL_STRINGIZE(S)  _STL_STRINGIZE_(S)
 
-// Note that _STL_PRAGMA is load-bearing;
-// it still needs to exist even once CUDA and ICC support _Pragma.
-#if defined(__CUDACC__) || defined(__INTEL_COMPILER)
-#define _STL_PRAGMA(PRAGMA) __pragma(PRAGMA)
-#else
-#define _STL_PRAGMA(PRAGMA) _Pragma(#PRAGMA)
-#endif
-
+#define _STL_PRAGMA(PRAGMA)          _Pragma(#PRAGMA)

```

Comment:
nit - why the large spacing? Shouldn't it be:

```suggestion
#define _STL_PRAGMA(PRAGMA) _Pragma(#PRAGMA)
```

Reply:
This is the effect of [`AlignConsecutiveMacros: Consecutive`](https://releases.llvm.org/19.1.0/tools/clang/docs/ClangFormatStyleOptions.html#alignconsecutivemacros):

https://github.com/microsoft/STL/blob/0d8f517ae3828284fed594741b847db940167a59/.clang-format#L53